recipe/meta.yaml

```diff
@@ -1,27 +1,29 @@
-{% set version = "3.2.1" %}
+{% set version = "3.2.3" %}
 
 package:
   name: graphql-core
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/g/graphql-core/graphql-core-{{ version }}.tar.gz
-  sha256: 9d1bf141427b7d54be944587c8349df791ce60ade2e3cccaf9c56368c133c201
+  sha256: 06d2aad0ac723e35b1cb47885d3e5c45e956a53bc1b209a9fc5369007fe46676
 
 build:
-  noarch: python
+  skip: True  # [py<36]
   number: 0
   script:
     - python -c '__import__("os").unlink("pyproject.toml")'
     - {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
+    - poetry
   run:
-    - python >=3.6
-    - typing_extensions >=4
+    - python
+    - typing_extensions >=4,<5
```

Comment:
This needs to be at least version 4.4 and is only needed below python 3.10: https://github.com/graphql-python/graphql-core/blob/main/pyproject.toml#L45

Reply:
I will need to update `typing_extensions` to 4.4